fladgejt/webui/predmety.py

```diff
@@ -21,8 +22,10 @@ def get_informacny_list(self, kod_predmetu, akademicky_rok):
             app.d.akRokComboBox.options, title=akademicky_rok)
         app.d.akRokComboBox.select(index)
```

Comment:
Mám podozrenie, že toto nebude fungovať, ak naposledy niekto hľadal predmet aj podľa iných kritérií (povedzme názov) a tento predmet ich nespĺňa. Zostanú vyplnené vo formulári. Asi tu treba volať `__query_dialog`, ktorý všetky nepoužité polia vyprázdňuje.